frontend/src/component/layout/MainLayout/DraftBanner/DraftBanner.tsx

```diff
@@ -40,9 +40,11 @@ const StyledDraftBanner = styled(Box)(({ theme }) => ({
     paddingRight: theme.spacing(9),
     marginLeft: 'auto',
     marginRight: 'auto',
-    [theme.breakpoints.down(1024)]: {
+    [theme.breakpoints.down('lg')]: {
```

Comment:
<img width="576" alt="Screenshot 2024-05-27 at 13 14 41" src="https://github.com/Unleash/unleash/assets/1394682/adfce11e-fa57-4828-bbf6-7ba5ff7d5dbd">
fixing this part of UI